docs/sanic/exceptions.md

```diff
@@ -43,3 +43,11 @@ Some of the most useful exceptions are presented below:
   usually occurs if there is an exception raised in user code.
 
 See the `sanic.exceptions` module for the full list of exceptions to throw.
+
+You can also use `exceptions_base`, it can catch the exception just like
+normal catching. It can be used for catching all unexpected error and eliminate
+the risk that server information being stolen by attackers/
```

Comment:
yes, inheriting ErrorHandler and overwrite the `lookup` function can achieve the same, but rather than the user to directly touch the ErrorHandler Interface, which may be changed in the future, I hope to have built in support for this important security-related issue and can be directly used by every user. I do handle other minor issue through using CustomHandler. I just think it may be good to let all the user to have this one as nature support.